libusb/os/windows_winusb.c

```diff
@@ -2454,29 +2454,17 @@ static int winusbx_open(int sub_api, struct libusb_device_handle *dev_handle)
 {
 	struct winusb_device_priv *priv = usbi_get_device_priv(dev_handle->dev);
 	struct winusb_device_handle_priv *handle_priv = get_winusb_device_handle_priv(dev_handle);
-	HANDLE file_handle;
 	int i;
 
 	CHECK_WINUSBX_AVAILABLE(sub_api);
+	(void)dev_handle;
```

Comment:
We have an UNUSED macro, if that is the point here.

Reply:
Fixed

Reply:
Ideally put this no-op statement above the other check that actually does something.